src/Components/StepSummaryItem.tsx

```diff
@@ -25,11 +26,15 @@ const LockIconPositioner = styled(Flex)`
 export const StepSummaryItem: React.SFC<StepSummaryItemProps> = ({
   title,
   onChange,
+  orderSource,
   children,
   locked,
   ...others
 }) => {
   const showHeading = title || (onChange && !locked)
+  const allowChangeShipping =
+    !locked && onChange && orderSource !== "private_sale"
```

Comment:
Looks like the `locked` flag works well for our use case. Instead of expanding the logic in the `StepSummaryItem`, I wonder if we can pass in `locked` for private sale orders. It will display the lock icon, and I think it makes sense.

Reply:
ah cool, thanks for pointing this out, I didn't know in which cases `locked` would be true. I'll update the component.